src/sentry/services/hybrid_cloud/notifications/service.py

```diff
@@ -78,7 +79,7 @@ def get_subscriptions_for_projects(
         user_id: int,
         project_ids: list[int],
         type: NotificationSettingEnum,
-    ) -> Mapping[int, tuple[bool, bool, bool]]:
+    ) -> Mapping[int, RpcGroupSubscriptionStatus]:
```

Comment:
This looks like a breaking change as well. We would need to stage this out over multiple deploys to prevent errors.